views/comp_avatar.ejs

```diff
@@ -0,0 +1,128 @@
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-sm">
+            <span class="avatar-icon-sm avatar-icon-secondary"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-md">
+            <span class="avatar-icon-md avatar-icon-success"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-lg">
+            <span class="avatar-icon-lg avatar-icon-caution"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-xl">
+            <span class="avatar-icon-xl avatar-icon-error"></span>
+        </div>
+    </div>
+    <div class="component__container">
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-round-xs">
+            <span class="avatar-icon-xs avatar-icon-primary"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-round-sm">
+            <span class="avatar-icon-sm avatar-icon-secondary"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-round-md">
+            <span class="avatar-icon-md avatar-icon-success"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-round-lg">
+            <span class="avatar-icon-lg avatar-icon-caution"></span>
+        </div>
+        <div class="avatar-status">
+            <img src="https://picsum.photos/72" alt="" class="avatar-round-xl">
+            <span class="avatar-icon-xl avatar-icon-error"></span>
+        </div>
+    </div>
+    <pre><code class="language-html">&lt;div class=&quot;avatar-status&quot;&gt;
+    &lt;img src=&quot;https://picsum.photos/72&quot; alt=&quot;&quot; class=&quot;avatar-xs&quot;&gt;
+    &lt;span class=&quot;avatar-icon-xs avatar-icon-primary&quot;&gt;&lt;/span&gt;
+&lt;/div&gt;
+&lt;div class=&quot;avatar-status&quot;&gt;
+    &lt;img src=&quot;https://picsum.photos/72&quot; alt=&quot;&quot; class=&quot;avatar-round-xs&quot;&gt;
+    &lt;span class=&quot;avatar-icon-xs avatar-icon-primary&quot;&gt;&lt;/span&gt;
+&lt;/div&gt;</code></pre>
+    <hr>
+    <h2 id="avatar-with-text">Avatar with text</h2>
+    <div class="component__container">
+        <div class="avatar-text avatar-xs">XS</div>
+        <div class="avatar-text avatar-sm">SM</div>
+        <div class="avatar-text avatar-md">MD</div>
+        <div class="avatar-text avatar-lg">LG</div>
+        <div class="avatar-text avatar-xl">XL</div>
+    </div>
+    <pre><code class="language-html">&lt;div class=&quot;avatar-text avatar-sm&quot;&gt;SM&lt;/div&gt;</code></pre>
+    </article>
+    </section>
```

Comment:
I cannot find opening tag of article tag

Reply:
it's inside head.ejs partial to keep the repetition to minimum.